app/src/main/java/org/greenstand/android/TreeTracker/models/StepCounter.kt

```diff
@@ -0,0 +1,63 @@
+package org.greenstand.android.TreeTracker.models

```

Comment:
This is the class that incorporates your suggestion to isolate the step count collection in its own class.